README.md

```diff
@@ -183,6 +183,12 @@ class {'chocolatey':
 }
 ~~~
 
+#### Install chocolatey using a proxy server
+
+~~~puppte
```

Comment:
This should be `puppet` for the syntax to highlight correctly.

Reply:
fixed